DXRCore/DeusEx/Classes/DXRActorsBase.uc

```diff
@@ -1483,3 +1483,13 @@ function DebugMarkKeyPosition(vector pos, coerce string id)
     a = Spawn(class'DXRGoalMarker',,,pos);
     DebugMarkKeyActor(a, id);
 }
+
+static function LogInventory(Actor actor)
+{
+    local Inventory item;
+
+    log(actor.class @ actor.bindname $ " inventory:");
```

Comment:
would be good to log the actual actor name, not sure you really need the class since there's only a few cases where you can't determine the class name from the actor name (any classname that ends with a number loses the number when making the actor name)

```suggestion
    log(actor @ actor.class @ actor.bindname $ " inventory:");
```